.eslintrc

```diff
@@ -202,6 +202,7 @@
     // The following must be off so that we can enable
     // "@typescript-eslint/semi", which checks semicolons in TS files.
     "semi": "off",
+    "semi-spacing": "error",
```

Comment:
We do not have any errors with this in our codebase?

Reply:
No, we do not have any errors. Further checked if check is working correctly by introducing an error and it caught that successfully.

Reply:
That is great!